CONTRIBUTING.md

```diff
@@ -163,3 +173,180 @@ describe 'a dog', ->
 # Returns `undefined`.
 disablePackage: (name, options, callback) ->
 ```
+
+## Issue and Pull Request Labels
+
+This section lists the labels we use to help us track and manage issues and pull requests. Most labels are used across the core `atom/atom` repository and all package repositories, but some are specific for `atom/atom`.
```

Comment:
How about `Most labels are used across all atom repositories, but some are specific to atom/atom`?